src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -73,13 +74,17 @@ public function getFunctions()
      */
     public function load($uuid)
     {
-        $contentStructure = $this->contentMapper->load(
-            $uuid,
-            $this->requestAnalyzer->getWebspace()->getKey(),
-            $this->requestAnalyzer->getCurrentLocalization()->getLocale()
-        );
+        try {
+            $contentStructure = $this->contentMapper->load(
+                $uuid,
+                $this->requestAnalyzer->getWebspace()->getKey(),
+                $this->requestAnalyzer->getCurrentLocalization()->getLocale()
+            );
 
-        return $this->structureResolver->resolve($contentStructure);
+            return $this->structureResolver->resolve($contentStructure);
+        } catch (DocumentNotFoundException $e) {
+            return;
```

Comment:
we should avoid suppressing the exception (at least completely). @danrot @chirimoya what do you think? should we only log the exception or should we rethrow it in dev environment? The same question we should ask for all the twig extension